controllers/devfileregistry_controller_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright Red Hat
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"github.com/devfile/registry-operator/api/v1alpha1"
+	. "github.com/devfile/registry-operator/pkg/test"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// The following test is only intended to test creation and updates to the DevfileRegistry CR, while ignoring the fact that routes will fail because they will
+// not resolve to an actual URL
+
+var _ = Describe("DevfileRegistry controller test", func() {
+	Context("When Creating and updating a DevfileRegistry CR with valid values", func() {
+		It("Should create and update a DevfileRegistry CR Name with Status URL not found", func() {
+			Expect(k8sClient.Create(ctx, getDevfileRegistryCR(devfileRegistryName, devfileRegistriesNamespace,
+				image))).Should(Succeed())
+			drlLookupKey := types.NamespacedName{Name: devfileRegistryName, Namespace: devfileRegistriesNamespace}
+			dr := &v1alpha1.DevfileRegistry{}
+			Eventually(func() error {
+				return k8sClient.Get(ctx, drlLookupKey, dr)
+			}, Timeout, Interval).Should(Succeed())
+
+			Expect(dr.Name).Should(Equal(devfileRegistryName))
+			Expect(dr.Status.URL).Should(Equal("")) // an empty URL is an indicator that a URL was not resolved
```

Comment:
The fake Route CRD I used in HAS, should have a status field added to it, where you can manually set the host name? This would unblock the rest of the tests, and you could verify that the proper host name gets set in the status and the deployments that get created

Reply:
I'm actually going to delete this in favour of the integration-tests...sorry, I'll request a re-review when I push updates